releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -91,6 +91,8 @@ private NotesBuilder() {
      * @return a map which represents release notes.
      * @throws IOException if an I/O error occurs.
      * @throws GitAPIException if an error occurs when accessing Git API.
+     * @noinspection UseOfSystemOutOrSystemErr
+     * @noinspectionreason UseOfSystemOutOrSystemErr - used for CLI output
```

Comment:
https://github.com/checkstyle/contribution/pull/561#discussion_r671886128
I am also not sure, @romani, @pbludov can you check it?

Reply:
Let's just suppress it for now, since we have a GSOC project around this (release) and it would be better not to make changes in case some script depends on this output (though, I doubt it matters).